### ISSUE TITLE
DSE throws exception on btr

### ISSUE THREAD

**Issue by robert-yates** [JonathanSalwan/Triton#766]
The following code produces an exception

```
BITS 64
SECTION .text


mov r14, 1234567812345678h
btr     r14d, 19h
ret
```

> ast.cpp
```
      if (this->children[0]->getBitvectorSize() != this->children[1]->getBitvectorSize())
        throw triton::exceptions::Ast("BvsmodNode::init(): Must take two nodes of same size.");
```

**Comment by JonathanSalwan** [comment]
Mmmmh, I think it's fixed with the last `dev-v0.7` branch. What is your build number?

**Comment by robert-yates** [comment]
confirmed: its working with `dev-v0.7` i was using `master`

/R.
